cypress/e2e/pages/owners.pages.js

```diff
@@ -109,6 +111,21 @@ export function openRemoveOwnerWindow(btn) {
   cy.get('div').contains(removeOwnerStr).should('exist')
 }
 
+export function getAddressToBeRemoved() {
```

Comment:
I would rather avoid using this method by few reasons: a) it takes always first value, and b) it does not use controlled value. All test values must be controlled and predictable. This is because if there is a bug and the value is incorrect, but the method is it is now just takes the first value no matter what, we will get poor assertion. 
Please add data-testid to the address value field, put address value in constants (if it is not there already), and assert that the address for deletion is what we expect.